network_visualizer/src/window.py

```diff
@@ -0,0 +1,19 @@
+from PyQt5.QtWidgets import *
+from PyQt5.QtCore import *
+from PyQt5.QtGui import *
```

Comment:
Since we're using `python_qt_bindings`, these lines should all be:
```
from python_qt_binding.QtWidgets import *
from python_qt_binding.QtCore import *
from python_qt_binding.QtGui import *
```

Reply:
Let's also avoid * imports. The only thing we need from here is
`from python_qt_binding.QtWidgets import QMainWindow`